leaderboard/api/tests/IntegrationTests/SentinelWebApplicationFactory.cs

```diff
@@ -29,7 +29,8 @@ public class CustomWebApplicationFactory<TStartup>
                     logging.AddConsole();
                     logging.AddDebug();
                 })
-                .UseStartup<IntegrationTests.Startup>();
+                .UseStartup<IntegrationTests.Startup>()
```

Comment:
Opening up the project it seems to work fine.  However, when I build I get a ton of warnings:

```
"..\openhack-devops-proctor\leaderboard\api\tests\IntegrationTests\IntegrationTests.csproj" (Build target) (1:7) ->
       (ResolveAssemblyReferences target) ->
         C:\Program Files\dotnet\sdk\2.1.302\Microsoft.Common.CurrentVersion.targets(2106,5): warning MSB3277: Found conflicts between different versions of "Microsoft.Extensions.Logging.Abstractions" that could not be resolved.  These reference conflicts are listed in the build log when log verbosity is set to detailed. [..\openhack-devops-proctor\leaderboard\api\tests\IntegrationTests\IntegrationTests.csproj]
```